specs/resolvers/cluster-resolvers.spec

```diff
@@ -0,0 +1,33 @@
+PIPELINES-23
+# Cluster resolvers spec
+
+## Checking the functionality of cluster resolvers: PIPELINES-23-TC01
+Steps:
+    * Validate Operator should be installed
+    * Create project "resolver-test-tasks"
+    * Create
+      |S.NO|resource_dir                                                |
+      |----|------------------------------------------------------------|
+      |1   |testdata/resolvers/tasks/resolver-test-task.yaml            |
+      |2   |testdata/resolvers/tasks/resolver-test-task2.yaml           |
+    * Create project "resolver-test-pipelines"
+    * Create
+      |S.NO|resource_dir                                                |
+      |----|------------------------------------------------------------|
+      |1   |testdata/resolvers/pipelines/resolver-test-pipeline.yaml    |
+    * Create project "resolver-test-pipelineruns"
+    * Create
+      |S.NO|resource_dir                                                              |
+      |----|--------------------------------------------------------------------------|
+      |1   |testdata/resolvers/pipelineruns/resolver-test-pipelinerun.yaml            |
+      |2   |testdata/resolvers/pipelines/resolver-test-pipeline-same-ns.yaml          |
+      |3   |testdata/resolvers/pipelineruns/resolver-test-pipelinerun-same-ns.yaml    |
```

Comment:
IIUIC you are testing two scenarios. Feel free to split them so that they can run separately. You can use pre-conditions for parts that are common like in https://github.com/openshift-pipelines/release-tests/blob/master/specs/operator/auto-prune.spec#L6-L7